pattern-2/tile/tile.yml

```diff
@@ -0,0 +1,321 @@
+      label: com.mysql.jdbc.Driver
+      default: true
+    - name: com.microsoft.sqlserver.jdbc.SQLServerDriver
+      label: com.microsoft.sqlserver.jdbc.SQLServerDriver
+  - name: conf_reg_db_query
+    type: string
+    label: Validation Query
+  - name: conf_reg_db_username
+    type: string
+    label: Username
+  - name: conf_reg_db_password
+    type: secret
+    label: Password
+- name: identity_db
+  label: Identity Datasource connection information
+  description: WSO2 Identity Server - Identity Datasource connection information
+  properties:
+  - name: identity_db_jdbc_url
+    type: string
+    label: JDBC URL
+  - name: identity_db_driver
+    type: dropdown_select
+    label: Driver Class Name
+    options:
+    - name: com.mysql.jdbc.Driver
+      label: com.mysql.jdbc.Driver
+      default: true
+    - name: com.microsoft.sqlserver.jdbc.SQLServerDriver
+      label: com.microsoft.sqlserver.jdbc.SQLServerDriver
+  - name: identity_db_query
+    type: string
+    label: Validation Query
+  - name: identity_db_username
+    type: string
+    label: Username
+  - name: identity_db_password
+    type: secret
+    label: Password
+- name: analytics_db
+  label: Identity Server Analytics Datasource connection information
+  description: WSO2 Identity Server - Identity Server Analytics Datasource connection information
+  properties:
+  - name: analytics_db_jdbc_url
+    type: string
+    label: JDBC URL
+  - name: analytics_db_driver
+    type: dropdown_select
+    label: Driver Class Name
+    options:
+    - name: com.mysql.jdbc.Driver
+      label: com.mysql.jdbc.Driver
+      default: true
+    - name: com.microsoft.sqlserver.jdbc.SQLServerDriver
+      label: com.microsoft.sqlserver.jdbc.SQLServerDriver
+  - name: analytics_db_query
+    type: string
+    label: Validation Query
+  - name: analytics_db_username
+    type: string
+    label: Username
+  - name: analytics_db_password
+    type: secret
+    label: Password
+
+packages:
+- name: routing_release
+  type: bosh-release
+  path: routing-0.178.0.tgz
+
+- name: wso2is
+  type: bosh-release
+  path: wso2is-5.7.0-bosh-release.tgz
+  jobs:
+  - name: nfs_server
+    templates:
+    - name: nfs_server
+      release: wso2is-release
+    memory: 4096
+    ephemeral_disk: 4096
+    persistent_disk: 16384
+    cpu: 2
+    static_ip: 1
+
+  - name: wso2is_analytics_worker
+    instances: 2
+    templates:
+    - name: wso2is_analytics_worker
+      release: wso2is-release
+      provides:
+        conn: {shared: true}
+      properties: {}
+    - name: health_check
+      release: wso2is-release
+      colocated: true
+      post_deploy: true
+      run_default: on
+      label: IS Health Check
+      description: Checks if Identity Server Analytics Worker is up and running
+    memory: 4096
+    ephemeral_disk: 4096
+    persistent_disk: 16384
+    cpu: 2
+    static_ip: 1
+    max_in_flight: 1
+    properties:
+      health_check:
+        endpoint: "tcp://localhost:7612"
```

Comment:
Health check does not work for this TCP endpoint.

Reply:
fixed with 3d673e1